cds_ils/importer/documents/importer.py

```diff
@@ -284,9 +307,10 @@ def validate_found_matches(self, not_validated_matches):
                 import_doc_title.lower()
 
             valid_identifiers = self._validate_match_identifiers(document)
+            valid_serial_volumes = self._validate_volumes(document)
 
             if titles_not_equal or editions_not_equal or pub_year_not_equal \
-                    or not valid_identifiers:
+                    or not valid_identifiers or not valid_serial_volumes:
```

Comment:
I would twist the conditions to have `titles_equal and editions_equal...` to improve readability and avoid the negation...